keyboards/keebio/levinson/keymaps/ksamborski/keymap.c

```diff
@@ -0,0 +1,151 @@
+ */
+    [_RS] = LAYOUT_ortho_4x12(
+        KC_COMM , KC_1   , KC_2   , KC_3   , KC_PLUS, KC_MINS, KC_LCBR, KC_RCBR, KC_LPRN, KC_RPRN, _______, _______,
+        KC_DOT  , KC_4   , KC_5   , KC_6   , KC_HASH, KC_ASTR, KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  ,
+        KC_LSFT , KC_7   , KC_8   , KC_9   , KC_SLSH, KC_EQL , KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 ,
+        _______ , _______, KC_0   , _______, _______, _______, _______, _______, _______, _______, _______, _______
+        ),
+
+/* Lower
+ * ,-----------------------------------------.    ,-----------------------------------------.
+ * |  `   |  !   |  @   |  #   |  $   |  %   |    |  Home|  PgDn|  PgUp|  End |  (   |  )   |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      | Vol+ | Mute | Play |      | Ins  |    |  Left|  Down|  Up  | Right|  {   |  }   |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      | Vol- | < M  |  M > |      |      |    |  ^   |  &   |  *   |  '   |  [   |  ]   |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      |      |      |      |LOWER |      |    | Shift|Adjust|      |      |      |      |
+ * `-----------------------------------------'    `-----------------------------------------'
+ */
+    [_LW] = LAYOUT_ortho_4x12(
+        KC_GRV  , KC_EXLM, KC_AT  , KC_HASH, KC_DLR , KC_PERC,    KC_HOME , KC_PGDN, KC_PGUP, KC_END  , KC_LPRN, KC_RPRN,
+        _______ , KC_VOLU, KC_MUTE, KC_MPLY, _______, KC_INS ,    KC_LEFT , KC_DOWN, KC_UP  , KC_RIGHT, KC_LCBR, KC_RCBR,
+        _______ , KC_VOLD, KC_MPRV, KC_MNXT, _______, _______,    KC_CIRC , KC_AMPR, KC_ASTR, KC_QUOT , KC_LBRC, KC_RBRC ,
+        _______ , _______, _______, _______, _______, _______,    KC_LSFT , _______, _______, _______ , _______, _______
+        ),
+
+/* Adjust
+ * ,-----------------------------------------.    ,-----------------------------------------.
+ * | Reset|      |  Up  |      |      | Rec1 |    | Rec2 | Plain| Snake|Gradnt|      | Del  |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * | Caps | Left | Down | Right|MkItPk| Play1|    | Play2| RGB  | HUE+ | SAT+ | BRI+ |      |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      |      |      |      |      | Stop1|    | Stop2| MODE | HUE- | SAT- | BRI- |      |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |    |      |      |      |      |      |      |
+ * `-----------------------------------------'    `-----------------------------------------'
+ */
+    [_AD] = LAYOUT_ortho_4x12(
+        RESET  , XXXXXXX,  KC_UP , XXXXXXX, XXXXXXX, DM_REC1 ,    DM_REC2 , RGB_M_P, RGB_M_SN,RGB_M_G, XXXXXXX, KC_DEL ,
```

Comment:
You had a nice, pretty alignment going originally; why not keep it that way? :slightly_smiling_face: 

```suggestion
        RESET  , XXXXXXX,  KC_UP , XXXXXXX, XXXXXXX, DM_REC1,    DM_REC2 , RGB_M_P, RGB_M_SN,RGB_M_G, XXXXXXX, KC_DEL ,
```